CHANGELOG.md

```diff
@@ -2,7 +2,68 @@
 
 ## New Features:
 
-No changes to highlight.
+### Theme Sharing 🎨 🤝
+
+You can now share your gradio themes with the world!
+
+After creating a theme, you can upload it to the HuggingFace Hub to let others view it, use it, and build off of it!
+
+### Uploading
+There are two ways to upload a theme, via the theme class instance or the command line.
+
+1. Via the class instance
+```python
+my_theme.to_hub(repo_name="my_theme",
```

Comment:
Should we rename to `push_to_hub()` to make it consistent with [other HF libraries](https://huggingface.co/docs/transformers/v4.26.1/en/main_classes/trainer#transformers.Trainer.push_to_hub)?

Reply:
Agreed